packages/SwingSet/src/kernel/state/storageWrapper.js

```diff
@@ -1,7 +1,7 @@
 import { assert } from '@agoric/assert';
 import { insistStorageAPI } from '../../lib/storageAPI.js';
 
-// We manage a host-realm Storage object with a has/getKeys/get/set/del API.
+// We manage a host-realm Storage object with a has/getKeys/get/set/delete API.
```

Comment:
We no longer use multiple Realms, so let's say "we wrap a host-provided Storage object with ..." and remove the bit about cross-realm contamination.